client/src/components/memeShowPage.js

```diff
@@ -21,10 +29,17 @@ const MemeShowPage = (props) => {
         getMeme()
     }, [])
 
+    const reviewTileComponents = meme.reviews.map((reviewObject) => {
+        return <ReviewTile key={reviewObject.id} {...reviewObject} />
+      })
+
     return(
         <>
             <h1>{meme.title}</h1>
             <img src={meme.memeUrl}/>
+            <div>
+                <ul>{reviewTileComponents}</ul>                
+            </div>
```

Comment:
this div is not doing anything. remove it!

Reply:
Done